.github/workflows/noDeploy.yml

```
@@ -1,42 +1,109 @@
-        run: npm run astro-check
+      # Parallel builds for better performance
+      - name: Build shared package
+        run: npm run build:shared:ci &
 
-      - name: Run unit tests
-        run: npm run test:unit
+      - name: Build frontend
+        run: npm run build:frontend &
+
+      - name: Wait for builds to complete
+        run: wait 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)